src/engine/cuecontrol.cpp

```diff
@@ -854,15 +832,15 @@ void CueControl::cueDefault(double v) {
 void CueControl::pause(double v) {
     QMutexLocker lock(&m_mutex);
     //qDebug() << "CueControl::pause()" << v;
-    if (v != 0.0) {
+    if (v != 0) {
```

Comment:
Integer Comparison will be wrong since the fractional part of the double is truncated. The CPU has to convert the integer to double inside its FPU anyway. 

A great link! Thank you very much. This would have same me some time. 
The differences between clang and gcc is, that clang uses the SSE2 extension by default.

I will revert for readability.